solr/contrib/ltr/src/java/org/apache/solr/ltr/rest/ManagedModelStore.java

```diff
@@ -116,13 +131,7 @@ private Feature parseFeature(Map<String,Object> featureMap,
       throw new FeatureException("feature " + name
           + " not found in store " + featureStore.getName());
     }
-
-    final Object normObj = featureMap.get(CommonLTRParams.FEATURE_NORM);
-    if (normObj != null) {
-      final Normalizer norm = Normalizer.fromMap(solrResourceLoader,
-          (Map<String,Object>) normObj);
-      meta = new FilterFeature(meta, norm);
-    }
+    meta = new FilterFeature(meta);
```

Comment:
Do we need to make a new FilterFeature, or can we just use the featureStore feature directly now?